src/Command/I18nExtractCommand.php

```diff
@@ -231,7 +231,7 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
             $message = "What is the path you would like to output?\n[Q]uit";
             $localePaths = array_values(App::path('locales'));
             if (!$localePaths) {
-                $localePaths[] = ROOT . 'resources' . DIRECTORY_SEPARATOR . 'locales';
+                $localePaths[] = ROOT . DS . 'resources' . DIRECTORY_SEPARATOR . 'locales' . DIRECTORY_SEPARATOR;
```

Comment:
```suggestion
                $localePaths[] = ROOT . DIRECTORY_SEPARATOR . 'resources' . DIRECTORY_SEPARATOR . 'locales' . DIRECTORY_SEPARATOR;
```

Mixing `DS` and `DIRECTORY_SEPARATOR` isn't nice.